examples/medplum-nextjs-demo/app/page.tsx

```diff
@@ -1,13 +1,16 @@
+'use client';
```

Comment:
We should eventually try to see if we can support React Server Components, but this is sensible for now because of how browser-specific our `MedplumClient` is.